_data-prepper/expression-syntax.md

```diff
@@ -0,0 +1,193 @@
+---
+layout: default
+title: Expression syntax
+nav_order: 12
+---
+
+# Supported operators
```

Comment:
I think the title of the page is supposed to match the first heading - but now I'm not 100% sure.